### ISSUE TITLE
[Feature] Pass through expansion of giver ID's

### ISSUE THREAD

**Issue by ghthor** [vise890/giver#1]
Add either an option to enable or make the default behavior that giver expands the ID's that it can find and ignores everything else. This would enable mixing and matching of giver ID's within the arguments to a command.

```sh
# Given an alias such as
alias G='$(giver give | giver pick $@!)'

# This would enable something like
vim G 1-3 new.txt
```

**Comment by vise890** [comment]
totally resolved in [a90d896b799daf2bd2d037a620a610e372dffca9](https://github.com/vise890/giver/commit/a90d896b799daf2bd2d037a620a610e372dffca9). you can now `--barf` non ID things back out

**Comment by vise890** [comment]
dammit, i'm gonna have to start implementing this thing..
